src/ASM/LR/ASMu3Dmx.C

```diff
@@ -532,14 +534,18 @@ bool ASMu3Dmx::integrate (Integrand& integrand,
             // Compute Hessian of coordinate mapping and 2nd order derivatives
             if (integrand.getIntegrandType() & Integrand::SECOND_DERIVATIVES) {
               if (!utl::Hessian(Hess,fe.hess(geoBasis),Jac,Xnod,
-                                d2Nxdu2[geoBasis-1],dNxdu[geoBasis-1]))
-                return false;
+                                d2Nxdu2[geoBasis-1],dNxdu[geoBasis-1])) {
+                ok = false;
+                continue;
+              }
 
               for (size_t b = 0; b < m_basis.size() && ok; ++b)
                 if ((int)b != geoBasis)
                   if (!utl::Hessian(Hess,fe.hess(b+1),Jac,Xnod,
-                                    d2Nxdu2[b],fe.grad(b+1),false))
-                    return false;
+                                    d2Nxdu2[b],fe.grad(b+1),false)) {
+                    ok = false;
+                    continue;
```

Comment:
At least here the `continue` is not needed, as we are at the end of the loop-block it affects anyway. Compare with the ASMs3Dmx-method. It is less confusing if they have similar logic.

Reply:
removed.